science/orthanc-dicomweb/Makefile

```diff
@@ -0,0 +1,64 @@
+# Created by: maintainer.freebsd@xpoundit.com
+
+PORTNAME=	orthanc-dicomweb
+DISTVERSION=	1.6
```

Comment:
I think we can just use `PORTVERSION` here.

Reply:
Ah, in my understanding, isn't `DISTVERSION` preferred over `PORTVERSION` (though in this case they are the same anyways)?